docs/resources/advanced_cluster.md

```diff
@@ -533,6 +533,7 @@ replication_specs {
 }
 ```
 
+* `external_id` - Unique 24-hexadecimal digit string that identifies the replication object for a shard in a Cluster. When using the new sahrding configuration, this corresponds to Shard ID displayed in the UI.
```

Comment:
```suggestion
* `external_id` - Unique 24-hexadecimal digit string that identifies the replication object for a shard in a Cluster. This value corresponds to Shard ID displayed in the UI. When using old sharding configuration (replication spec with `num_shards` greater than 1) this value is not populated.
```
Small rephrasing suggestion to clarify this always corresponds to Shard ID, but is not populated if using old sharding configuration.